src/Analysis/Ast/Impl/Modules/BuiltinsPythonModule.cs

```diff
@@ -158,7 +159,17 @@ protected override string[] GetScrapeArguments(IPythonInterpreter interpreter)
             Analysis.SpecializeFunction("max", BuiltinsSpecializations.Identity);
             Analysis.SpecializeFunction("min", BuiltinsSpecializations.Identity);
             Analysis.SpecializeFunction("next", BuiltinsSpecializations.Next);
-            Analysis.SpecializeFunction("open", BuiltinsSpecializations.Open, new[] { "io" });
+
+            Analysis.SpecializeFunction("open", BuiltinsSpecializations.Open, new[] {
```

Comment:
Is it the same set of parameters for all supported python versions?

Reply:
Nope, Python 3 has all of these, but Python 2 only has `open(name[, mode[, buffering]])`. Probably needs to be conditionally specialized.